tests/Custom/TimeoutTest.php

```diff
@@ -76,6 +79,32 @@ public function testShortPageLoadTimeoutThrowsException(): void
 
         $this->expectException(DriverException::class);
         $this->expectExceptionMessage('Page failed to load: ');
+
         $session->visit($this->pathTo('/page_load.php?sleep=2'));
     }
+
+    /**
+     * @group legacy
```

Comment:
The symfony deprecations helper requires the test to be in the "legacy" group, so I could not just alter the existing test.

Reply:
OK.